src/fr/composants/pied-de-page/design.md

```diff
@@ -8,17 +8,21 @@ date: "git Last Modified"
 
 ## Structure du pied de page — éléments recommandés
 
-La **bande de liens du pied de page** GC s'étend sur le bas de la page. Elle contient le mot-symbole officiel du gouvernement du Canada et des liens vers les conditions générales et les avis de confidentialité des produits, ainsi que des liens facultatifs vers les pages Canada.ca pour les médias sociaux, les applications, ainsi qu'une section « à propos ».
-
-Le **mot-symbole Canada** est l'identificateur mondial du gouvernement du Canada et l'un des trois symboles officiels de l'identité du gouvernement du Canada.
+<ol class="anatomy-list">
+  <li>La <strong>bande de liens du pied de page</strong> GC s'étend sur le bas de la page. Elle contient le mot-symbole officiel du gouvernement du Canada et des liens vers les conditions générales et les avis de confidentialité des produits, ainsi que des liens facultatifs vers les pages Canada.ca pour les médias sociaux, les applications, ainsi qu'une section « à propos ».</li>
+  <li>Le <strong>mot-symbole Canada</strong> est l'identificateur mondial du gouvernement du Canada et l'un des trois symboles officiels de l'identité du gouvernement du Canada.</li>
+</ol>
 
 <img class="b-sm b-default p-400" src="/images/fr/components/anatomy/gcds-footer-anatomy-recommended.svg" alt="Website footer taxonomy with two sections. The first section is the Footer links bands in a light grey banner with some links like Social media, Mobile applications, About Canada.ca and so on. The second section is the Canada workmark indicating Canada logo." />
```

Comment:
I know this isn't related to the work you did in this PR, but just noticing that this `img` has English alt text on the French design page.

Reply:
Good catch! Asked translation to translate it and they will send it back tomorrow.